Sources/Search.php

```diff
@@ -931,7 +931,7 @@ function PlushSearch2()
 
 	// *** Encode all search params
 
-	// All search params have been checked, let's compile them to a single string... made less simple by PHP 4.3.9 and below.
```

Comment:
Essentially what this is saying is that this is complex, and originally it was more complex than this, a decade ago, to fix bugs in 4.3.x, but the facet that made it specifically tricky is no longer applicable.